src/main/java/week6/ParkingLotTest.java

```diff
@@ -0,0 +1,151 @@
+package week6;
+
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.stream.IntStream;
+import org.junit.Test;
+
+public class ParkingLotTest {
+
+
+  @Test
+  public void testEmpty() {
+    assertEquals("", ParkingLot.rearrangeCars(null, null).toString());
+    assertEquals("", ParkingLot.rearrangeCars(null, new int[1]).toString());
+    assertEquals("", ParkingLot.rearrangeCars(new int[1], null).toString());
+    assertEquals("", ParkingLot.rearrangeCars(new int[1], new int[2]).toString());
+    assertEquals("", ParkingLot.rearrangeCars(new int[0], new int[0]).toString());
+  }
```

Comment:
"empty" seems incorrect to me for most of these. These are invalid inputs.

I'd only call "ParkingLot.rearrangeCars(new int[0], new int[0])" empty input.